Formula/doxygen.rb

```diff
@@ -23,7 +23,7 @@ class Doxygen < Formula
   depends_on "cmake" => :build
   depends_on "graphviz" => :optional
   depends_on "qt5" => :optional
-  depends_on "llvm" => "with-clang" if build.with? "libclang"
+  depends_on "llvm" if build.with? "libclang"
```

Comment:
None at all. Also https://github.com/maxim-belkin/homebrew-core/blob/c714f17b2d1b65a5c2f84be855718ebf81288b96/Formula/doxygen.rb#L31 would need to be updated to `if build.with? "llvm"`.